python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_execute_pipeline.py

```diff
@@ -592,23 +601,30 @@ def test_step_failure(self, graphql_context):
         assert step_run_log_entry["error"]
         assert step_run_log_entry["level"] == "ERROR"
 
-        causes = step_run_log_entry["error"]["causes"]
+        assert step_run_log_entry["failureMetadata"]
+        assert step_run_log_entry["failureMetadata"]["metadataEntries"] == [
+            {
+                "__typename": "BoolMetadataEntry",
+                "label": "top_level",
+                "description": None,
+                "boolValue": True,
+            }
+        ]
 
-        assert len(causes) == 3
+        causes = step_run_log_entry["error"]["causes"]
+        assert len(causes) == 2
         assert [cause["message"] for cause in causes] == [
-            "Exception: Even more outer exception\n",
```

Comment:
why is this being deleted / do we expect this to change?

Reply:
special handling of `Failure`